python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -35,6 +35,52 @@
 import lsst.meas.algorithms as measAlg
 import lsst.meas.algorithms.utils as maUtils
 import lsst.meas.extensions.psfex as psfex
+from lsst.pipe.base import AlgorithmError
+
+
+class PsfexTooFewGoodStarsAlgorithmError(AlgorithmError):
```

Comment:
I think you can just call this `PsfexTooFewGoodStarsError`: no need to repeat the parent class "AlgorithmError" in the name.

Reply:
Good idea!